components/appmgt/org.wso2.carbon.appmgt.gateway/src/main/java/org/wso2/carbon/appmgt/gateway/handlers/common/SynapsePropertiesHandler.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Check the copyright format. 
"Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved."